modules/qi_manage.php

```diff
@@ -44,6 +44,7 @@ public function __construct()
 					$current_item = $settings->get_boards_dir() . $item;
 
 					// Need to get the dbname from the board.
+					$dbhost = $dbport = $dbname = $dbuser = $dbpasswd = $dbms = '';
 					@include($current_item . '/config.php');
```

Comment:
`@include($current_item . '/config.php');`
what about...
```
if (file_exists($current_item . '/config.php'))
{
    include($current_item . '/config.php');
}
```

Reply:
I don't know why that is like that and I tend not to mess with stuff like that because it's probably got some voodoo nonsense - due to there being code here dating back to 2007 - that would just break something somewhere else, that you never would have expected, given how disorganized the code here is.

Reply:
Hmm, having PHP 8 in mind and reading that "logic" I would go that why though, whatever you like is ok.